doc/sphinx/addendum/universe-polymorphism.rst

```diff
@@ -285,53 +283,58 @@ Insufficiently restrictive variance annotations lead to errors:
       Inductive Irrelevent@{*u} : Type@{u}.
 
       Section Universes.
-        Universe u v.
-        Constraint u < v.
-
-        (** An invariant universe blocks cumulativity from upper or lower levels. *)
-        Axiom inv_u : Invariant@{u}.
-        Axiom inv_v : Invariant@{v}.
-        Fail    Check (inv_u : Invariant@{v}).
-        Fail    Check (inv_v : Invariant@{u}).
-
-        (** A covariant universe allows cumulativity from a lower level. *)
-        Axiom co_u : Covariant@{u}.
-        Axiom co_v : Covariant@{v}.
-        Succeed Check (co_u : Covariant@{v}).
-        Fail    Check (co_v : Covariant@{u}).
-
-        (** An invariant universe allows cumulativity from any level *)
-        Axiom irr_u : Irrelevent@{u}.
-        Axiom irr_v : Irrelevent@{v}.
-        Succeed Check (irr_u : Irrelevent@{v}).
-        Succeed Check (irr_v : Irrelevent@{u}).
+        Universe low high.
+        Constraint low < high.
+
+        (* An invariant universe blocks cumulativity from upper or lower levels. *)
+        Axiom inv_low  : Invariant@{low}.
+        Axiom inv_high : Invariant@{high}.
+   .. coqtop:: all
```

Comment:
You need a blank line after `.. coqtop::`.  Several places.

Reply:
That won't cause a blank line in the example right?

Reply:
No, but you have no choice.  It doesn't work without the blank line.  BTW, a lot quicker if you build the refman locally (`make -f Makefile.dune refman-html`) rather than rely on CI.